scripts/install_third_party.sh

```diff
@@ -117,7 +117,7 @@ echo Checking if webtest is installed in third_party
 if [ ! -d "$TOOLS_DIR/webtest-1.4.2" ]; then
   echo Installing webtest framework
   # Note that the github URL redirects, so we pass in -L to tell curl to follow the redirect.
-  curl --silent -L https://github.com/Pylons/webtest/archive/1.4.2.zip -o webtest-download.zip
+  curl webtest-download.zip https://github.com/Pylons/webtest/archive/1.4.2.zip
```

Comment:
Er -- I'm not sure about this. It's printing a lot of random output to the terminal. Don't you need an -o flag?

Ditto elsewhere.